features/exploredash/build.gradle

```diff
@@ -56,6 +57,11 @@ hilt {
     enableAggregatingTask = true
 }
 
+ktfmt {
+    kotlinLangStyle()
+    maxWidth.set(120)
```

Comment:
The code style is based on kotlin language code style, except larger line boundary. I think a max line of 100 characters is less necessary nowadays and hinders readability in some cases.